assets/javascripts/discourse/components/chat-live-pane.js

```diff
@@ -937,19 +959,79 @@ export default Component.extend({
     return this.messages.findIndex((m) => m.id === message.id);
   },
 
-  @action
-  onChannelTitleClick() {
+  _goToChatableUrl() {
     if (this.chatChannel.chatable_url) {
       return this.router.transitionTo(this.chatChannel.chatable_url);
     }
   },
 
+  @action
+  onChannelTitleClick() {
+    return this._goToChatableUrl();
+  },
+
+  @action
+  quoteMessages() {
+    ajax(getURL(`/chat/${this.chatChannel.id}/quote.json`), {
```

Comment:
I don't think we need to test this but we probably want to handle any potential errors.